docs/token-transfer.md

```diff
@@ -0,0 +1,102 @@
+# Transferring Hbar Using the Hedera JSON-RPC Relay
+
+For illustration purposes, Metamask is used but this guide should be applicable to any JSON-RPC enabled wallet.
+
+## Add Hedera Network to Metamask
+
+1. Open Metamask and click on the circle in the upper right corner
```

Comment:
In this doc no need to repeat the others.
Either 
1. Use the same doc just add a separate section
2. Reference the HBAR doc in here.

My preference would be for a single doc with different scripts where applicable

Reply:
what do you think of the current form?